Products/Archetypes/skins/archetypes/widgets/js/textcount.js

```diff
@@ -1,12 +1,13 @@
 <!-- Original:  Ronnie T. Moore -->
 <!-- Dynamic 'fix' by: Nannette Thacker -->
```

Comment:
Since you are touching this file anyway, could you replace these html comments by javascript comments?